container/docker/handler.go

```diff
@@ -253,11 +252,16 @@ func newDockerContainerHandler(
 
 	// split env vars to get metadata map.
 	for _, exposedEnv := range metadataEnvs {
+		if exposedEnv == "" {
```

Comment:
I don't understand this part.  Is this fixing a bug?

Reply:
```golang
func main() {
	a := ""
	b := "test"
	if strings.Contains(b, a) {
		fmt.Println("yes")
	}
}
```

if like before, it's fine for totally match. But if matched with `strings.Contains`, `""` may be wrong